app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def initialize(r, message)
 
   def redirect_to_back(default = root_url)
     if !request.env["HTTP_REFERER"].blank? and request.env["HTTP_REFERER"] != request.env["REQUEST_URI"]
-      redirect_to :back
+      redirect_back fallback_location: root_path
     else
       redirect_to default
     end
```

Comment:
this method actually do the same thing as `redirect_back fallback_location: root_path` do (except default path of course). I suggest to search by `redirect_to_back` method & replace to a standard rails 5 method `redirect_back ...`

Reply:
method that I've suggested is more DRY, and it's name fully describes it's function